repl/scala-2.11/src/main/scala/org/apache/spark/repl/Main.scala

```diff
@@ -44,7 +44,9 @@ object Main extends Logging {
 
   private def scalaOptionError(msg: String): Unit = {
     hasErrors = true
+    //scalastyle:off
```

Comment:
In the IDE it was showing me error hence I put those there , if you say we can remove it